MAUI/ProgressBar/Appearance.md

```diff
@@ -0,0 +1,392 @@
+linearProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("DD20A7DB"), Value = 50 });
+linearProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("FF1C96C5"), Value = 75 });
+this.Content = linearProgressBar;
+
+SfCircularProgressBar circularProgressBar = new SfCircularProgressBar();
+circularProgressBar.Progress = 100;
+circularProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("88A0D9EF"), Value = 0 });
+circularProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("AA62C1E5"), Value = 25 });
+circularProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("DD20A7DB"), Value = 50 });
+circularProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("FF1C96C5"), Value = 75 });
+this.Content = circularProgressBar;
+
+{% endhighlight %}
+
+{% endtabs %} 
+
+![gradient](images/appearance/gradient.png)
+
+## Thickness
+
+### Linear progress bar
+
+In the linear progress bar, the height of the track, progress and secondary progress can be customized using the `TrackHeight` and `ProgressHeight` and `SecondaryProgressHeight` properties, respectively.
+
+{% tabs %} 
+
+{% highlight xaml %}
+
+<progressBar:SfLinearProgressBar Progress="100" 
+                                 TrackHeight="10" 
+                                 ProgressHeight="10"
+                                 SecondaryProgressHeight="10">
+
+</progressBar:SfLinearProgressBar>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfLinearProgressBar linearProgressBar = new SfLinearProgressBar();
+linearProgressBar.Progress = 100;
+linearProgressBar.TrackHeight = 10;
+linearProgressBar.ProgressHeight = 10;
+linearProgressBar.SecondaryProgressHeight = 10;
+this.Content = linearProgressBar;
+
+{% endhighlight %}
+
+{% endtabs %} 
+
+![thickness_linear](images/appearance/thickness.png)
+
+### Circular progress bar
+
+The following properties are used to customize the appearance of the circular progress bar:
+
+* `ProgressRadiusFactor`: Defines the outer radius of the progress indicator.
+* `ProgressThickness`: Defines the thickness of the progress indicator.
+* `TrackRadiusFactor`: Defines the outer radius of the track indicator.
+* `TrackThickness`: Defines the thickness of the track indicator.
+* `ThicknessUnit`: Specifies whether the `ProgressThickness` or `TrackThickness` are defined in pixel or factor.
+
+The `ProgressThickness` or `TrackThickness` of the progress bar can be specified either in pixel or factor. If the `ThicknessUnit` is specified as Pixel, then the range will be rendered based on the provided pixel value. If the `ThicknessUnit` is set as factor, the provided factor value will be multiplied with outer radius. For example, if the thickness width is set as 0.1, then 10% of outer radius is considered as thickness.
```

Comment:
The `ProgressThickness` or `TrackThickness` of the progress bar can be specified either in pixel or factor. If the `ThicknessUnit` is specified as Pixel, the range will be rendered based on the provided pixel value. If the `ThicknessUnit` is set as a factor, the provided factor value will be multiplied by the outer radius. For example, if the thickness width is set as 0.1, then 10% of the outer radius is considered as thickness.

Reply:
done